### ISSUE TITLE
Remove Third-Party IP from Repo

### ISSUE THREAD

**Issue by dennisss** [dennisss/tansa#91]
***DO NOT DO UNTIL CLOSER TO THE PERFORMANCE***

We can do this when everyone is in a single room and all branches have been merged into master, otherwise this will nuke everyone's work.

Eventually need to use this (https://help.github.com/articles/remove-sensitive-data/) to remove all data files or choreography translated from Parsons Dance materials. We don't have the right to distribute that creative property, so it has to be removed before open-sourcing this repo.

This must be done before the performance.
